src/item/thumbnail/hooks.ts

```diff
@@ -40,17 +40,35 @@ export const useItemThumbnail =
   };
 
 // create a new thumbnail hook because of key content
+/**
+ * Fetch item's thumbnail url
+ * @param id corresponding item's id for thumbnail, used to retrieve cache data and know if the item has a thumbnail
+ * @param item corresponding item for thumbnail, used to know if the item has a thumbnail
+ * @param size size of the thumbnail we want
+ * @returns url of the thumbnail given size
+ */
 export const useItemThumbnailUrl =
   (queryConfig: QueryClientConfig) =>
-  ({ id, size = DEFAULT_THUMBNAIL_SIZE }: { id?: UUID; size?: string }) => {
+  ({
+    id,
```

Comment:
I understand that it will break the existing code, but do we still need the `id` ? we could pass the id through the item ?

Reply:
right! But I will avoid to break, also you don't always need to pass the item (most of the time you do) but going through the id only should be possible too I think. But it's true you can use only the item.